### PR TITLE
Update Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by ralfralfsen**
Minor updates/corrections around REST API connectivity.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ Here are some extra things to think about as you implement Microsoft Teams in yo
...
 
-- For presence status, Microsoft Teams must check whether the mailbox is hosted on Exchange Online or on-premises. The service then decides where to access the mailbox. To enable the Teams service to check the mailbox location through the REST API call to the Exchange Online service, you have to deploy an Exchange hybrid environment by running the Exchange Hybrid Configuration wizard, as described in [Create a hybrid deployment with the Hybrid Configuration wizard](/exchange/hybrid-deployment/deploy-hybrid).
+- For presence status, Microsoft Teams must check whether the mailbox is hosted on Exchange Online or on-premises. The service then decides where to access the mailbox. To enable the Teams service to check the mailbox location through the REST API call. Please note that presence updates are pulled each 60min for Exchange on-premises hosted mailoboxes.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:125]:
```suggestion
- For presence status, Microsoft Teams must check whether the mailbox is hosted on Exchange Online or on-premises. The service then decides where to access the mailbox to enable the Teams service to check the mailbox location through the REST API call. Please note that presence updates are pulled every 60 minutes for Exchange on-premises hosted mailboxes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ralfralfsen, kindly commit the suggested changes if you agree with them. Many thanks.

File: Teams/Exchange-Teams-interact.md (hunk 1 above)
Comment (ralfralfsen) [reply Teams/Exchange-Teams-interact.md:125]: This statement is actually incorrect: "To enable the Teams service to check the mailbox location through the REST API call"
The mailbox location determined using AutodiscoverV2. Please rephrase the bullet point to:


"For the presence status, Microsoft Teams must check whether the mailbox is hosted on Exchange Online or on-premises. This is determined by doing a AutodiscoverV2 request for REST, the service then decides where to access the mailbox. Please note that presence updates are pulled each 60min for Exchange on-premises hosted mailboxes."

File: Teams/Exchange-Teams-interact.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ If mailboxes are hosted on-premises, to create and view meetings, the following
 
 - Mailboxes are hosted in Exchange Server 2016 Cumulative Update 3 or later.
 
-- Autodiscover and Exchange Web Services is published externally.
+- Autodiscover, Exchange Web Services (EWS) and REST API (for presence status) virtual directories are published externally.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:93]:
```suggestion
- Autodiscover, Exchange Web Services (EWS), and REST API (for presence status) virtual directories are published externally.
```

File: Teams/Exchange-Teams-interact.md (hunk 1 above)
Comment (JohanFreelancer9) [reply Teams/Exchange-Teams-interact.md:125]: Thank you for the explanation. I have updated my suggestion accordingly.

Comment (ralfralfsen) [reply Teams/Exchange-Teams-interact.md:125]: very good, thanks!

**Review by JohanFreelancer9 — COMMENTED** [review]
@ralfralfsen - Could I please ask you to click on the commit buttons to commit these changes? Many thanks.

**Comment by ralfralfsen** [comment]
I don't have a Commit button. I just have a "close pullrequest".

Ralf

**Comment by JohanFreelancer9** [comment]
@ralfralfsen - you should be able to merge the suggestions either by using the [Commit suggestion] button in the suggestions, or directly editing the file accordingly and pushing the changes to this PR branch to resolve the added suggestions.

**Comment by ralfralfsen** [comment]
@JohanFreelancer9 can you please check if it worked now?

**Comment by JohanFreelancer9** [comment]
@ralfralfsen - no, the changes have not been committed.  @scanum - can you please ask for this PR to be merged with my suggested changes. Many thanks.

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
Closing because of age, inactivity, file conflicts, and article updates have made these changes no longer applicable.

cc: @scanum

**Pull request closed without merging**